tools/crm_mon.c

```diff
@@ -2005,14 +2012,13 @@ mon_refresh_display(gpointer user_data)
 #if CURSES_ENABLED
             blank_screen();
             print_status(out, mon_data_set, stonith_history, options.mon_ops, show,
-                         options.neg_location_prefix, options.only_node);
-            refresh();
+                         options.neg_location_prefix, options.only_node, options.only_rsc);
```

Comment:
It's been too long since I looked at this patch to tell whether this was intentional or not.  refresh gets called from a lot of other places, like basically every time a printing function in crm_mon_curses.c gets called.  So it's probably not strictly necessary, but it probably doesn't hurt anything either.  I'll put it back just to keep the number of inexplicable unrelated changes in a single commit to a minimum.

Reply:
The difference seems to be that, with the refresh in place, the cursor is at the start of the next line instead of at the end of the previous line.